chpl/chpl-service/src/test/java/gov/healthit/chpl/dao/impl/DeveloperDaoTest.java

```diff
@@ -251,4 +253,32 @@ public void createDeveloperAcbMap() {
 		SecurityContextHolder.getContext().setAuthentication(null);
 	}
 	
+	/**
+	 * Given that I am authenticated to CHPL as any user

```

Comment:
I don't see anything here that suggests you're actually authenticated to the CHPL. Maybe the Given should be something like: "Given the CHPL is accepting search requests"